openhcl/virt_mshv_vtl/src/cvm_cpuid/mod.rs

```diff
@@ -178,8 +179,11 @@ pub struct ParsedCpuidEntry {
 }
 
 /// Prepares and caches the results that should be returned for hardware CVMs.
+#[derive(Inspect)]
 pub struct CpuidResults {
```

Comment:
I still want this whole thing merged with the non-CVM cpuid handling. We shouldn't have two completely different table formats.

But anyway, that's certainly out of scope for this change.

Reply:
I agree - it's really annoying that they're completely different.